api/dna_api.go

```diff
@@ -144,13 +144,15 @@ func (api *DnaApi) ActivateInvite(ctx context.Context, args ActivateInviteArgs)
 		}
 		from = crypto.PubkeyToAddress(key.PublicKey)
 	}
-	payload := api.baseApi.secStore.GetPubKey()
-	to := args.To
-	if to == nil {
-		coinbase := api.baseApi.getCurrentCoinbase()
-		to = &coinbase
+
+	var pubKey []byte
+	if args.PubKey != nil {
+		pubKey = *args.PubKey
+	} else {
+		pubKey = api.baseApi.secStore.GetPubKey()
 	}
-	hash, err := api.baseApi.sendTx(ctx, from, to, types.ActivationTx, decimal.Zero, decimal.Zero, decimal.Zero, args.Nonce, args.Epoch, payload, key)
+	to, err := crypto.PubKeyBytesToAddress(pubKey)
```

Comment:
Please, check `err` here in order to prevent request panic

Reply:
Done. 👍🏻